draft-ietf-httpbis-message-signatures.md

```diff
@@ -434,7 +434,7 @@ If this parameter is included with a component identifier, the component value M
 2. For each field value in the set:
     0. Strip leading and trailing whitespace from the field value. Note that since HTTP field values are not allowed to contain leading and trailing whitespace, this will be a no-op in a compliant implementation.
     1. Remove any obsolete line-folding within the line and replace it with a single space (" "), as discussed in {{Section 5.2 of HTTP1}}. Note that this behavior is specific to {{HTTP1}} and does not apply to other versions of the HTTP specification.
-    2. Encode the bytes of the resulting field value's ASCII representation as a Byte Sequence.
+    2. Encode the bytes of the resulting field value's bytes as a Byte Sequence. Note that most fields are restricted to ASCII characters, but other octets could be included and need to be treated with care.
```

Comment:
What does "encode as a byte sequence" mean here? (Sounds like an identity transformation).

Reply:
Byte Sequence is a data type from structured fields, so this is saying take the bytes and put them in that data structure.

Reply:
But that's a breaking change which also invalidates all examples, no?

Reply:
Hu? https://www.ietf.org/archive/id/draft-ietf-httpbis-message-signatures-16.html#section-2.5-12 has:

~~~
"@method": POST
"@authority": example.com
"@path": /foo
"content-digest": sha-512=:WZDPaVn/7XgHaAy8pmojAkGWoRx2UFChF41A2svX\
  +TaPm+AbwAgBWnrIiYllu7BNNyealdVLvRwEmTHWXvJwew==:
"content-length": 18
"content-type": application/json
"@signature-params": ("@method" "@authority" "@path" \
  "content-digest" "content-length" "content-type")\
  ;created=1618884473;keyid="test-key-rsa-pss"~~~

Reply:
None of those use the `bs` flag which is the subject of this section.

Reply:
Well, in that case you need to clearly point out thar field values that use non-ASCII can only be used *with* the "bs" flag (otherwise what happens? an error?)

Reply:
That is incorrect. They are encoded with percent encoding as you suggested above.